docs/README.md

```diff
@@ -76,13 +76,21 @@ Examples:
 kn revision list --service srvc # CHECK this since current command does not have --service flag
 ```
 
+### Plugins
+
+Kn supports plugins, which allow you to extend the functionality of your Kn installation with custom commands as well as shared commands that are not part of the core distribution of Kn. 
+
+Plugins follow a similar architecture to [kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) with some small differences. One key difference is that Kn plugins can either live in your `PATH` or in a chosen and specified directory. [Kn plugins](plugins.md) shows how to install and create new plugins as well as giving some examples and best practices.
+
```

Comment:
Are we planning to add content in plugins.md ?

Reply:
yup, that's the next PR. Held up because I am juggling a lot of stuff but will find time to push this by early next week along with examples.